code/game/objects/items/RTD.dm

```diff
@@ -246,7 +246,7 @@
 
 					//infer available overlays on the floor to recreate them to the best extent
 					QDEL_LIST(design_overlays)
-					var/floor_designs = floor.managed_overlays
+					floor_designs = floor.managed_overlays
```

Comment:
just make a new list. we shouldn't be reusing the old one here

Reply:
Also do me a favor and type floor_designs as a list in this file. it's not but it really should be